projects/frontend-ui/shared/README.md

```diff
@@ -0,0 +1,17 @@
+# VDK Shared Components library
+
+This project implements common UI functionality that is used in the data-pipelines project.
+Everything in this folder is project agnostic and there is nothing data-pipelines specific about them.
+The hope with having them in a separate directory is that other projects might leverage them.
+At the moment there are many other private projects in VMware which are using them.
```

Comment:
Don't think it's needed to mention who uses it if not some public artifacts, it's out of scope of the public repository;

Reply:
I think it give legitimacy to the decision. But happy to remove it also.